README.md

```diff
@@ -121,6 +121,12 @@ You "_may use special characters_", but only some of them - and we won't necessa
 ![Mindware](/screenshots/mindware1.png "Mindware")
 ![Mindware](/screenshots/mindware2.png "Mindware")
 
+### [MKB NetBankár](https://www.mkbnetbankar.hu/)
+
+![MKB NetBankár](/screenshots/mkb.png "MKB NetBankár")
+
+It only accepts lowercase letters, uppercase letters and numbers (any other character counts as forbidden character).
```

Comment:
Thanks for your contribution! Please put the comment above the screenshot like the other listings.